mods/disable-virtual-desktop-transition.wh.cpp

```diff
@@ -0,0 +1,205 @@
+// ==WindhawkMod==
+// @id              disable-virtual-desktop-transition
+// @name            Disable Virtual Desktop Transition Animation
+// @description     Disables the animation when switching between virtual desktops on Windows 10.
+// @version         1.0
+// @author          Isabella Lulamoon (kawapure)
+// @github          https://github.com/kawapure
+// @twitter         https://twitter.com/kawaipure
+// @homepage        https://kawapure.github.io
+// @include         dwm.exe
+// @compilerOptions -lkernel32
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+# Disable Virtual Desktop Transition Animation
+
+This mod disables the animation seen when switching between virtual desktops in Windows 10.
+
+In order to use this mod, you have to hook into `dwm.exe`. In order to do this, you must go into Windhawk advanced settings
+and remove the exclusion entry for `<critical_system_processes>`.
```

Comment:
Allowing injection into all critical processes isn't necessary, it should be enough to allow `dwm.exe`. You can copy the message from here:

https://github.com/ramensoftware/windhawk-mods/blob/10dd219bec0369013eeeb9a3c0bee7efc3987f41/mods/disable-rounded-corners.wh.cpp#L34-L40